backend/endpoints/train_endpoints.py

```diff
@@ -70,8 +75,35 @@ def tabular_run():
         )
 
         try:
-            createTrainspaceData(tabular_data)
+            create_dynamo_item_from_obj("trainspace", tabular_data)
             print(id)
+
+            async def run():
```

Comment:
woah! I'm very confused exactly what this code does? Can you explain why we are using this `aiohttp` library? I'm worried if this seems a bit hacky? @dwu359

Reply:
Should work the same way as async/await in js, except there is more boilerplate code since it is python. The aiohttp library needs to be used to make the request without waiting for the response. If we used requests package instead for example, the backend would wait for a response from the training microservice before sending a response back to the user.